articles/article-063-reverse-engineering.md

```diff
@@ -0,0 +1,29 @@
+<!-- BEGIN TITLE -->
+# Reverse Engineering
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In general terms, *Reverse Engineering* describes the process of analyzing an existing apparatus to figure out how it works and how it was made.
+The same can be done for a computer program but instead of looking at screws and cogwheels we are looking at the machine instructions.
+These are effectively the 0's and 1's that tell the computer what to do.
+This is much harder to understand than the code that was used to generate the program.
+A skilled reverse engineer can still use this information to gain knowledge about the software. This can be used to detect security flaws, which can then either be reported to the programmers for them to eliminate or it can be exploited maliciously.
+
+<!-- END BODY -->
+
+![Reverse Engineering Flow](../images/image-063-reverse-engineering.jpeg)
+
+
+## Optional text
+<!-- BEGIN OPTIONAL -->
+Maybe explain graphic in text.
```

Comment:
Please don't put review/feedback notes here. Either add them outside the "optional" section, or as a comment to the PR.